### ISSUE TITLE
infoWeather not working on server

### ISSUE THREAD

**Issue by trueBugbat** [maruohon/minihud#361]
inforWeather is supposed to show the current weather, but I've noticed that it doesn't work on servers.  Is there something like the slime chunk thing that allows it to work, or does it not work on servers?

**Comment by sakura-ryoko** [comment]
Servers sends you if it's "raining" but they don't tell you when it's thundering, even though you can see it on the screen.

**Comment by trueBugbat** [comment]
It doesn't tell me when it's raining either, though.

![2024-09-18_22 44 16](https://github.com/user-attachments/assets/a4e7f3ee-5002-4576-815e-38ea99779eb4)

**Comment by sakura-ryoko** [comment]
Yeah, idk; ive seen the "raining" part show on a server without thundering before.  If it's "thundering" it might show clear also.

Clear is just the "Default" when the mod doesn't detect an actual Weather status.
